### ISSUE TITLE
Should be able to flag mines as definitely bad.

### ISSUE THREAD

**Issue by khipkin** [jQuarry/jQuarry#3]

**Comment by chrissprague** [comment]
@khipkin Yes, I think what we want here is a reminder/memo function which lets you label mines what you think they possibly can be. So not just "bad" but also if a mine could possibly have a point value of 1, 2, etc. :+1:
